<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VisafindMainWindow</class>
 <widget class="QMainWindow" name="VisafindMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>804</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>仪器发现与配置界面</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableWidget" name="visa_find_restab">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>431</width>
      <height>231</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>1</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>2</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>3</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>4</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>5</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>接口类型</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>接口地址</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>仪器类别</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>仪器详细信息</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string/>
     </property>
     <property name="flags">
      <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled|ItemIsTristate</set>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="visa_find_interface">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>40</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>查看所有接口</string>
    </property>
   </widget>
   <widget class="QComboBox" name="visa_interface_combox">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>40</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>ALL</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>USB</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>NET</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>GPIB</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>LXI</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>320</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
   <widget class="Phonon::VideoPlayer" name="videoPlayer">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>700</y>
      <width>300</width>
      <height>200</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="interinfo_psh">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>140</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>查询接口信息</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="interinfo_browser">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>180</y>
      <width>201</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>70</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>选择的地址</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="seladd_browser">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>90</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>804</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="file_Save_Visa"/>
    <addaction name="file_Save_Ivi"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>查询</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <action name="file_Save_Visa">
   <property name="text">
    <string>保存Visa发现的仪器</string>
   </property>
  </action>
  <action name="file_Save_Ivi">
   <property name="text">
    <string>保存Ivi发现的驱动模块</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Phonon::VideoPlayer</class>
   <extends>QWidget</extends>
   <header location="global">phonon/videoplayer.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>VisafindMainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>637</x>
     <y>285</y>
    </hint>
    <hint type="destinationlabel">
     <x>651</x>
     <y>337</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
